packages/utils/fluid-main/src/SharedObjectBase.ts

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export * from  "@fluidframework/shared-object-base";
```

Comment:
Only people that are going write DDS is interested in the shared-object-base, right?  Do we want to expose it here?

Reply:
I pulled the list of packages from the package.json's of our FluidExamples. Seems two of them have shared-object-base in their package.json, but don't actually use the package itself. I can remove it.

https://github.com/microsoft/FluidExamples/search?q=shared-object-base